packages/gatsby/cache-dir/static-entry.js

```diff
@@ -50,7 +51,9 @@ const createElement = React.createElement
 
 export default (pagePath, callback) => {
   let bodyHtml = ``
-  let headComponents = []
+  let headComponents = [
```

Comment:
I think this is OK?

This ends up putting it _after_ some things like style elements and stuff in the head tags, which is probably fine.

i.e.

```html
<style>
/* snipped */
</style>
<meta name="generator" content="Gatsby 2.0.0-rc.26"/>
<title data-react-helmet="true">Reproduction of #7590</title>
<meta data-react-helmet="true" name="description" content="Sample"/>
<meta data-react-helmet="true" name="keywords" content="sample, something"/>
<style type="text/css"></style>
```

Reply:
yeah, the order doesn't matter